app/generate.js

```diff
@@ -134,18 +160,20 @@ async function processOrg(orgId) {
     throw new Error('Template folder not found.');
   }
 
-  const headerTemplate = template(fs.readFileSync(`./templates/${orgData.theme}/header.tpl.html`, 'utf8'));
   const footerTemplate = template(fs.readFileSync(`./templates/${orgData.theme}/footer.tpl.html`, 'utf8'));
 
   if (!orgData.defaultHeader.logo) {
```

Comment:
This is essentially the same `if` as I commented in `generatePage` - which is why it's better to move it into `generateHeader`